cumulusci.yml

```diff
@@ -1171,6 +1181,18 @@ flows:
             2:
                 task: test_data_1k
 
+    regression_tests:
```

Comment:
a new flow to deploy contact merge testing and potentially all other tests that requires metadata that does not include in the package.